test/uk/gov/hmrc/mdgstub/controllers/MdgStubControllerControllerSpec.scala

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.mdgstub.controllers
+
+import java.io.InputStream
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.util.ByteString
+import play.api.http.{HeaderNames, Status}
+import play.api.mvc.RawBuffer
+import play.api.test.{FakeRequest, Helpers}
+import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
+
+import scala.concurrent.duration._
+
+class MdgStubControllerControllerSpec extends UnitSpec with WithFakeApplication {
+
```

Comment:
If this test is creating a fake application instance then it's bigger than a simple unit test and should live with integration tests under the /it directory.